src/analysis/vector/geometry_checker/qgsgeometrygapcheck.h

```diff
@@ -95,7 +95,9 @@ class ANALYSIS_EXPORT QgsGeometryGapCheck : public QgsGeometryCheck
     {
       MergeLongestEdge, //!< Merge the gap with the polygon with the longest shared edge.
       NoChange, //!< Do not handle the error.
-      AddToAllowedGaps
+      CreateNewFeature, //!< Create a new feature with the gap geometry
```

Comment:
API break due to new value for AddToAllowedGaps?

Reply:
I don't think this is public API or has the potential to break anything.
There are no python bindings for that and it's not stored anywhere outside memory.
Anyway it's a leftover from workarounds to previous API limitations, so this can be reverted.

Reply:
Ah, if it's not exposed to Python, then no issue!